variable/test/variable_bucket_non_owning_test.cpp

```diff
@@ -66,7 +66,7 @@ class VariableBucketNonOwningTypedTest : public VariableBucketNonOwningTest {
 using VariableBucketNonOwningTypes =
     ::testing::Types<VariableConstView, VariableView>;
 TYPED_TEST_SUITE(VariableBucketNonOwningTypedTest,
-                 VariableBucketNonOwningTypes);
+                 VariableBucketNonOwningTypes, );
```

Comment:
We have a redefined macro to avoid this warning. We should use that approach for consistency. https://github.com/scipp/scipp/blob/master/test/fix_typed_test_suite_warnings.h